build/Targets/Analyzers.Imports.targets

```diff
@@ -201,4 +205,11 @@
        ==================================================================================== -->
 
   <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != '' AND '$(ImportVSSDKTargets)' == 'true' AND '$(CIBuild)' == ''" />
+
+  <!-- The VSSDK immplements some targets which are run during design time builds per convention. 
+       If the Project is not a VSIX project and sets $CreateVSIXContainer=false these targets should be 
+       a no-op. However they dont check for that and that's a bug. To workaround, we override them
+       with empty targets and conditionally import these if CreateVSIXContainer is false.
+       Tracked by https://devdiv.visualstudio.com/DevDiv/_workitems?id=365685&fullScreen=false&_a=edit   -->
+  <Import Project="VSSDKDesignTimeBuildWorkaround.targets" Condition="'$(CreateVsixContainer)'=='false'" />
```

Comment:
I have copied this blindly from the Project system repo :). @srivatsn do you think this is still needed?